packages/notification-services-controller/src/NotificationServicesController/processors/process-notifications.test.ts

```diff
@@ -16,6 +17,12 @@ describe('process-notifications - processNotification()', () => {
     expect(result).toBeDefined();
```

Comment:
haha, yeah we were lazy with the tests here.

Future (on our team) - add better asserts here to ensure the expected type is correct.